fastcrypto/src/tests/ristretto255_tests.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'd be interested in extending this to the full set of RFC vectors, taking inspiration from https://github.com/pornin/crrl/blob/main/src/ristretto255.rs#L743

Reply:
Done!